debug/index.html

```diff
@@ -26,6 +32,96 @@
     hash: true
 });
 
+map.addControl(new mapboxgl.NavigationControl());
+map.addControl(new mapboxgl.GeolocateControl());
+map.addControl(new mapboxgl.ScaleControl());
+map.addControl(new mapboxgl.FullscreenControl());
```

Comment:
can you revert the changes on this file so it matches the `debug/index.html` file on master? you can add a new debug page for controls instead if you want too.